deploy/azure/app/kube/main.tf

```diff
@@ -15,7 +15,7 @@ module "default_label" {
 }
 
 module "app" {
-  source                               = "git::https://github.com/amido/stacks-terraform//azurerm/modules/azurerm-server-side-app?ref=v1.5.1"
+  source                               = "git::https://github.com/ensono/stacks-terraform//azurerm/modules/azurerm-server-side-app?ref=v1.5.1"
```

Comment:
Could you review this URL? It doesn't seem to work either

Reply:
the amido one doesn't work either

Reply:
its under the azure world.  Ill let Saul know and we can tackle it in #7695

Reply:
The URL is correct, terraform needs a double forward slash which browsers don't like

It should be upgraded to the latest version though which uses an alias A record instead of passing in the dns A record.